create-project-async/src/batched_project.py

```diff
@@ -0,0 +1,65 @@
+import json
+from src.graphql_document_creator import GraphQLDocumentCreator
+from src.graphql_utils import GraphQLUtils
+from src.helper import get_access_token
+from src.project import Project
+
+DEFAULT_BATCH_SIZE = 100
+
+
+class BatchedProject(Project):
+    def __init__(self, base_url: str, id: str, secret: str, documents_path: str, document_batch_size=DEFAULT_BATCH_SIZE):
+        if not 1 <= document_batch_size <= 100:
+            raise ValueError("document_batch_size must be between 1 and 100")
+
+        super().__init__(base_url, id, secret, documents_path)
+        self.document_batch_size = document_batch_size
+        self.graphql_utils = GraphQLUtils(base_url=self.base_url, headers=self.headers,
+                                          client_id=self.client_id, client_secret=self.client_secret)
+
+    def create(self, team_id: str, operations_path: str, name: str | None = None):
+        chunked_gql_documents = self.__get_chunked_gql_documents()
+
+        for index, gql_documents in enumerate(chunked_gql_documents):
+            operations = self._get_operations(
```

Comment:
1 yang aku notice, karena loggingnya sekarang muncul _setelah_ request selesai disubmit, ada masanya seolah-olah scriptnya "hang"

sepertinya bisa dikasih logging waktu batching documents / sebelum submit graphql requestnya, just for feedback sake

Reply:
ada retrieving access token sih ya 👀

Reply:
ohh!

ada uploading documents didepan semua ya

sepertinya itu bisa dilogging sedikit deh supaya ada feedback dan scriptnya ga hang gitu